### ISSUE TITLE
Custom tabs not focusable if none are active

### ISSUE THREAD

**Issue by jordanlapointe** [react-bootstrap/react-bootstrap#4737]
## Describe the bug

Tabs cannot be selected using keyboard navigation if none of the tab items are active. Issue caused by setting `tabindex="-1"` on inactive tabs, even if none are active.

## To Reproduce

1. Create a tabbed navigation without an `activeKey` or `defaultActiveKey` prop set
2. Press tab key to focus the navigation

## Reproducible Example

[Example](https://codesandbox.io/s/code-sandbox-examples-2nthn)

## Expected behavior

Should focus first tab within the tabbed navigation

## Environment

- MacOS 10.14
- Latest Chrome and Safari
- React-bootstrap 1.0.0-beta.12
